### ISSUE TITLE
Focus not visible with Shift-Tab Navigation

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#198]
Screengrab/videos (insert file names here)
2020 11 07, Focus not visible with Shift-Tab Navigation
[2020 11 07, Focus not visible with Shift-Tab Navigation.zip](https://github.com/CityOfBoston/Iterators/files/5514440/2020.11.07.Focus.not.visible.with.Shift-Tab.Navigation.zip)
.mp4

Link
https://www.boston.gov/my-neighborhood

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Safari

Trusted Tester Test

Brief Description

Steps to reproduce
1)	00:00, Open the link in a private browser window, enter a search and see a list of matches
2)	00:05, Use the tab key to navigate down the list.  Notice that the window scrolls in order to keep the focus visible on the screen.
3)	00:11, Use the shift-tab key combination to navigate backwards through the list.
4)	00:14, BUG:  The Focus is partially visible.
5)	00:17, Use the tab key to navigate further down the window.
6)	00:20, Use the shift-tab key combination to navigate backwards through the list.
7)	00:21, BUG:  The Focus is not visible as the window is navigated in reverse.

Desired Result
•	Focus is always visible during keyboard navigation

Actual Result
•	Focus is not visible when the keyboard navigation is done in reverse.

**Comment by reillyzlab** [comment]
This applies for all of the boxes (including the cards).

We should implement this.

**Comment by subaha-cob** [comment]
@david-iterators  UAT is now up with MNL tickets. 
https://d8-uat.boston.gov/

Try testing with this addresses only:

50 Milk St #2050 Boston, 02108
192 Dudley St Roxbury, 02119
27 Tyler St Boston, 02111
1 State St Ste 1500 Boston, 02108

**Comment by ollie-iterators** [comment]
This bug has been fixed
